package.json

```diff
@@ -86,7 +86,7 @@
     "gh-pages": "^1.2.0",
     "git-hooks-plus": "^1.0.1",
     "html-webpack-plugin": "^4.3.0",
-    "jest": "^26.4.2",
+    "jest": "^26.6.3",
```

Comment:
did you use this one to audit packages? https://www.npmjs.com/package/npm-check-updates

I don't think there is's only one out-of-date package